mock-exchange/exchange/exrates/transformer.go

```diff
@@ -0,0 +1,42 @@
+package exrates
+
+import (
+	"encoding/json"
+
+	"github.com/kowala-tech/kcoin/mock-exchange/app"
+)
+
+type Transformer struct {
```

Comment:
Should this not be an interface defined somewhere else?

```
type Transformer interface {
    Transform(request app.Request) (string, error)
}
```

and then there'd be an implementation for each exchange?

Reply:
Yes, you are right. I dont know why in the end this is here, I think I defined first in exchange folder. I am gonna returns this there, maybe I changed it when I got a cycle import problem.

Reply:
Ah wait! Check in exchange/

package exchange

import "github.com/kowala-tech/kcoin/mock-exchange/app"

type Transformer interface {
	Transform(request app.Request) (string, error)
}

Reply:
Exactly what you mentioned!